docs/conf/australia/2022/news/get-ready.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: Nov 28, 2022
+   :tags: {{year}}, {{shortcode}}-{{year}}, tickets
+
+Get Ready for the conference!
+=============================
+
+Hey everyone!
+
+We're just over a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.
```

Comment:
```suggestion
We're just over a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 2 days of content, collaboration, and conversation.
```

unless I'm mistaken?

Reply:
No, this is correct. Should be 2 days. Fixed it now.